deploy.cfg.example

```diff
@@ -27,11 +27,10 @@ auth-service-url = https://kbase.us/services/auth/api/legacy/KBase/Sessions/Logi
 # The KBase authorization service mkII url.
 auth2-service-url = https://kbase.us/services/auth
 
-# The urls for the Handle Service and Handle Manager.
+# The urls for the Handle Service.
 handle-service-url =
-handle-manager-url =
-# The token used to access the handle manager.
-handle-manager-token = 
+# The token used to access the handle manager (shock admin token).
```

Comment:
This is not necessarily a shock admin token - it's a token that allows the user to use the set_read_acl (or whatever, I don't remember the name exactly) method on the Handle Manager (now Handle Service). The Handle Service presumably has a token that *is* a shock admin, but this token may or may not be.

Reply:
👍